paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc

```diff
@@ -475,6 +483,122 @@ TEST(CpuQuantizePass, reshapeBetweenNonQuantizedOp) {
                   transpose_count, reshape_count, quant_count, dequant_count,
                   added_nodes_count, 2.0f * 127);
 }
+
+void MainTestCheckScales(
+    const ProgramDesc& prog,
+    const std::initializer_list<std::string> variable_names,
+    const std::string& var_without_scale) {
+  std::unique_ptr<ir::Graph> graph(new ir::Graph(prog));
+  int original_nodes_num, current_nodes_num;
+  std::stringstream ss;
+  ss << "Quantization scale for the variable " << var_without_scale
+     << " is missing.";
+  std::string error_msg = ss.str();
+  bool caught_exception = false;
+  try {
+    PreparePass(&graph, prog, variable_names, &original_nodes_num,
+                &current_nodes_num, var_without_scale);
+  } catch (paddle::platform::EnforceNotMet& error) {
+    caught_exception = true;
+    std::string ex_msg = error.what();
+    EXPECT_TRUE(ex_msg.find(error_msg) != std::string::npos);
+  }
+  EXPECT_TRUE(caught_exception);
+}
+
+// (a, w)->Conv->o,
```

Comment:
the `,` at the end is unnecessary here and in below comments :)

Reply:
Done.